src/Sylius/Component/Variation/Model/OptionValueTranslationInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c); Paweł Jędrzejewski
```

Comment:
I guess you have something wrong with your templates. This header shouldn't contain semicolon